File: Selenium/Activities/Activity_5.py

from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("The page title is: ", driver.title)
    act = webdriver.ActionChains(driver)
    ele = driver.find_element(By.ID, "D3Cube")
    ele.click()
    print(driver.find_element(By.CLASS_NAME, "active").text)
    act.double_click(ele).perform()
    print(driver.find_element(By.CLASS_NAME, "active").text)
    act.context_click(ele).perform()
    print(driver.find_element(By.CLASS_NAME, "active").text)
    driver.quit()

File: Selenium/Activities/Activity_5.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

import java.util.List;

public class Activity_5 {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver  = new ChromeDriver();
        Actions act = new Actions(driver);
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("The title of WebPage is: " + driver.getTitle());
        WebElement dice = driver.findElement(By.id("D3Cube"));
        dice.click();
        String text1 = driver.findElement(By.className("active")).getText();
        System.out.println(text1);
        Thread.sleep(1000);
        //double click
        act.doubleClick(dice).build().perform();
        String text2 = driver.findElement(By.className("active")).getText();
        System.out.println(text2);
        Thread.sleep(1000);
        //right click
        act.contextClick(dice).build().perform();
        Thread.sleep(1000);
        String text3 = driver.findElement(By.className("active")).getText();
        System.out.println(text3);
        Thread.sleep(1000);
        driver.quit();

    }
}
